addons/web/static/tests/views/fields/domain_field_tests.js

```diff
@@ -815,7 +818,7 @@ QUnit.module("Fields", (hooks) => {
         assert.verifySteps(["/web/webclient/load_menus"]);
 
         await doAction(webClient, 1);
-        assert.verifySteps(["/web/action/load", "get_views", "read", "search_count"]);
+        assert.verifySteps(["/web/action/load", "get_views", "read", "search_count", "fields_get"]);
```

Comment:
extra rpc?

Reply:
before, the view service did put the fields it gets from get_views in cache. Here we have to separated caches managed by 2 two different services. Gery told me that it's fine.